.circleci/config.yml

```diff
@@ -66,6 +66,10 @@ jobs:
                 test_name: "tomljson"
                 module: "github.com/pelletier/go-toml/cmd/tomljson"
                 allow_fail: <<parameters.allow_fail>>
+	    - run_test:
```

Comment:
Those tabs will need to be converted to spaces, to match the rest of the file.

Reply:
Yeah, I suppose go fmt will fix that?

Reply:
Nope, `go fmt` just runs on Go code, not YAML files.